src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj

```diff
@@ -235,8 +237,9 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="ProjectSystem\VS\PropertyPages\PropertyPageResources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
+      <Generator>PublicResXFileCodeGenerator</Generator>
```

Comment:
The XAML requires it since it has static bindings. The only reason the code worked before is someone had hand edited the generated CS file to make the class and members public. As soon as I made a resx file change, the dialog no longer worked.